CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# How to contribute
+
+Operator Marketplace is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, instructions on how to set up a dev environment, and other resources to help get contributions into operator-marketplace.
+
+# Email and Chat
+
+- Email: [operator-framework](https://groups.google.com/forum/#!forum/operator-framework)  
+
+## Setting up a dev environment
+
+- Fork the repository on GitHub
+- Clone the repository in your go path.
+- Get access to an Openshift cluster. The marketplace-operator is currently supported on Openshift 4.0. You can find instructions on installing an Openshift cluster [here](https://github.com/openshift/installer).
+- Turn off the Cluster-Service-Operator (CVO). The CVO manages the lifecycle of the marketplace-operator along with other internal Openshift components. Once the CVO is truned off, the marketplace-operator deployment can be deleted. 
+    `oc scale --replicas 0 -n openshift-cluster-version deployments/cluster-version-operator`
+- Delete the marketplace-operator deployment `oc delete deployment openshift-marketplace -n marketplace-operator`
+- Install [Operator-SDK](https://github.com/operator-framework/operator-sdk).
+- `cd` into the marketplace-operator repository and `dep ensure` to download all dependencies.
+- `operator-sdk up local --namespace=openshift-marketplace --kubeconfig=<path-to-kubeconfig-file>` to complie the marketplace-operator and start the operator in you dev environment.      
+
+## Reporting bugs and creating issues
+
+Reporting bugs is one of the best ways to contribute. If any part of the operator-marketplace project has bugs or documentation mistakes, please let us know by [opening an issue](https://github.com/operator-framework/operator-marketplace/issues/new)
```

Comment:
I suggest leaving out `Reporting bugs is one of the best ways to contribute.`

`opening an issue or a PR`